scenarios/intersections/4lane_t/scenario.py

```diff
@@ -42,7 +42,22 @@
     ),
 ]
 
-ego_missions = [t.EndlessMission(begin=("edge-south-SN", 1, 20))]
+ego_missions = [
+    t.EndlessMission(
+        begin=("edge-south-SN", 0, 20),
+        via=(
```

Comment:
these seem to be `checkpoints`, how do you feel about renaming?

Reply:
They are not required to be hit, just recommendations.  The policy determines any desire to hit them.